src/vsr/replica.zig

```diff
@@ -1070,6 +1141,13 @@ pub fn ReplicaType(
             assert(message.header.view == self.view);
             assert(message.header.replica == self.primary_index(message.header.view));
 
+            // Old/duplicate heartbeats don't count.
```

Comment:
add `assert(message.header.command == .commit)` at the start of the function

Reply:
Actaually, I think we don't do this for other functions at the moment, so might start doing that (eg, in `on_start_view_change`)